### DISCUSSION TITLE
Persistent Speedometer (toggle)

### DISCUSSION THREAD

**Discussion by ByteBitten** [organicmaps/organicmaps#1155] [Ideas]
When I don't need routing, I still like to see my speed on screen.
A settings toggle for making the speedometer persistent on screen to give the option.

**Comment by biodranik** [comment]
You can see your current speed now if you select your current position blue arrow.

**Reply by ByteBitten** [reply]
Ah, that is nice to know.
But it is very small.
My idea would be a decent sized one you could see at a glance while driving or biking.
Squinting for your speed is not that safe during travel :)

**Reply by biodranik** [reply]
Then we need to think about how to make it convenient for everyone, especially for those who don't need to see the speed.

**Reply by ByteBitten** [reply]
My idea for it:
- Settings toggle for Speedometer
if navigation = false
if speedoToggle = true
display speedometer

Showing it left under as hovering over the map would be my prefered location.

Waze has this option this way.
